ROADMAP.md

```diff
@@ -90,16 +90,58 @@ See details in [GitHub milestone v1.3.0](https://github.com/ratify-project/ratif
 
 See details in [GitHub milestone v1.4.0](https://github.com/ratify-project/ratify/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.4.0).
 
-### v2.x
+### v2.0.0-alpha.1
+**Status**: In procoess
+**Target date**: May 15, 2025
+
+**Major features**
+
+- Decouple Ratify core functionalities into a separate library([ratify-go](https://github.com/ratify-project/ratify-go)), which could be easily integrated into different projects.
+- Design new interfaces for `Verifier`, `Store`, and `PolicyEnforcer` to make them more extensible and flexible.
+- Support `notation-verifier` under [ratify-verifier-go](https://github.com/ratify-project/ratify-verifier-go) in terms of the new `Verifier` interface.
+- Support Registry Store and local OCI layout in terms of the new `Store` interface.
+- Support config based policy in terms of the new `PolicyEnforcer` interface.
+- Create new external data provider for Gatekeeper based on the new framework.
+
+### v2.0.0-alpha.2
+**Status**: Tentative
+**Target date**: June 30, 2025
+**Major features**
+- Support authentication with remote OCI registries.
+- Support key/certificate management for verifiers.
+
+### v2.0.0-beta.1
+**Status**: Tentative
+**Target date**: TBD
+**Major features**
+- Support cache management.
+- Improve performance of executor by supporting parallel execution of verifiers.
+
+### v2.0.0-beta.2
+**Status**: Tentative
+**Target date**: TBD
+**Major features**
+- Support more built-in verifiers to verify different artifact types.
+- Support CRDs for K8s users to configure Ratify add-on natively.
+- Support multi-tenancy.
 
-Status: Tentative
 
-Target date: TBD
+### v2.0.0-beta.3
+**Status**: Tentative
+**Target date**: TBD
+**Major features**
+- Build new ratify CLI tool with the basic functionalities.
+- Support instrumentation and telemetry.
```

Comment:
What's does instrumentation and telemetry refer here? Is it for collecting the Ratify Pod running metrics?

Reply:
Basically it will expose pods' metrics to monitor. Like integrating Prometheus to monitor request count and duration.